packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/form.tsx

```diff
@@ -86,9 +86,15 @@ const QuickStrategyForm = ({
                                     'quick-strategy__form-content--safari-fix': isSafari(),
                                 })}
                             >
-                                <div className='quick-strategy__title'>{localize('Quick strategy')}</div>
-                                <div className='quick-strategy__description'>
-                                    {localize('Choose a template and set your trade parameters.')}
+                                <div className='quick-strategy__text'>
+                                    <Text text='sm' weight='bold' line_height='s'>
```

Comment:
What does `text=sm` mean?

Reply:
@sara-fs should be `size` instead of `text`... i will change it, thanks that you notice

Reply:
fixed ✅